samples/bot-sharepoint-list/csharp/SPFx_Bot/Repositories/SharepointRepository.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Microsoft.BotBuilderSamples.SpfxBot.Models;
+using System.Net.Http;
+using System.IO;
+using System.Net;
+
+namespace Microsoft.BotBuilderSamples.SpfxBot.Repositories
+{
+    public class SharepointRepository
+    {
+        //This method is added as part of spfx changes. This method gets the authentication token
+        private static async Task<string> GetAuthenticationToken()
+        {
+            string tenantName = SettingsConfig.AppSetting("TenantName");
+            string tenantID = SettingsConfig.AppSetting("TenantID");
+            string resourceID = SettingsConfig.AppSetting("ResourceID");
+            string appClientID = SettingsConfig.AppSetting("AppClientID");
+            string appSecret = SettingsConfig.AppSetting("AppSecret");
+
+            string clientID = appClientID + "@" + tenantID;
+            string url = "https://accounts.accesscontrol.windows.net/tokens/OAuth/2";
+            string res = resourceID + "/" + tenantName + ".sharepoint.com@" + tenantID;
+            string body = $"grant_type=client_credentials&client_id={clientID}&client_secret={appSecret}&resource={res}";
+
+            HttpClient httpclient = new HttpClient();
+            var request = new HttpRequestMessage(HttpMethod.Post, url);
+            request.Headers.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
+            request.Content = (new StringContent(body, Encoding.UTF8, "application/x-www-form-urlencoded"));
+            HttpResponseMessage response = await httpclient.SendAsync(request);
+            string responseBody = await response.Content.ReadAsStringAsync();
+
+            if (!response.IsSuccessStatusCode)
+                throw new Exception(responseBody);
+
+            string accessToken = null;
+            accessToken = JsonConvert.DeserializeObject<SharepointRepository.TokenResponse>(responseBody).access_token;
+            return accessToken;
+        }
```

Comment:
Please add new line after function call. Check other classes and make same changes.